### DISCUSSION TITLE
CSS for table text-align (myst)

### DISCUSSION THREAD

**Discussion by chrisjsewell** [pradyunsg/furo#530] [Ideas] [answered]
Hey @pradyunsg 

As mentioned in: https://myst-parser.readthedocs.io/en/latest/syntax/syntax.html#tables,
myst adds css classes, to allow for text alignment of columns (not available in standard rST tables)

would you be able to add the requisite CSS in https://github.com/pradyunsg/furo/blob/main/src/furo/assets/styles/content/_tables.sass?

**Comment by pradyunsg** [comment, marked as answer]
Willing to file a PR for this?

**Reply by chrisjsewell** [reply]
Something like this https://github.com/pradyunsg/furo/pull/531?

**Reply by pradyunsg** [reply]
Yup.
